test/functional/ui/screen.lua

```diff
@@ -255,6 +255,17 @@ function Screen:set_default_attr_ids(attr_ids)
   self._default_attr_ids = attr_ids
 end
 
+function Screen:add_extra_attr_ids(extra_attr_ids)
```

Comment:
Nit (only mentioning since you are planning a followup): "extra" is redundant since it is implied by "add"

    add_attr_ids